modules/dnn/src/opencl/conv_layer_spatial.cl

```diff
@@ -49,18 +49,27 @@
 #if defined(FUSED_CONV_RELU)
 #define ACTIVATION_RELU_FUNCTION(x, c) ((Dtype)(x) > 0 ? (Dtype)(x) : ((Dtype)(x) * (Dtype)(negative_slope)))
 #define NEGATIVE_SLOPE_ARG Dtype negative_slope,
+#define RELU6_MAXVAL_ARG
 #elif defined(FUSED_CONV_PRELU)
 #define ACTIVATION_RELU_FUNCTION(x, c) ((Dtype)(x) > 0 ? (Dtype)(x) : ((Dtype)(x) * (Dtype)(negative_slope[c])))
 #define NEGATIVE_SLOPE_ARG __global const Dtype *negative_slope,
+#define RELU6_MAXVAL_ARG
 #elif defined(FUSED_CONV_POWER)
 #define ACTIVATION_RELU_FUNCTION(x, c) pow(x, power)
 #define NEGATIVE_SLOPE_ARG Dtype power,
+#define RELU6_MAXVAL_ARG
 #elif defined(FUSED_CONV_TANH)
 #define ACTIVATION_RELU_FUNCTION(x, c) tanh(x)
 #define NEGATIVE_SLOPE_ARG
+#define RELU6_MAXVAL_ARG
+#elif defined(FUSED_CONV_RELU6)
+#define ACTIVATION_RELU_FUNCTION(x, c) (clamp((Dtype)(x), min_value, max_value))
+#define NEGATIVE_SLOPE_ARG Dtype min_value,
+#define RELU6_MAXVAL_ARG Dtype max_value,
```

Comment:
```.diff
-#define NEGATIVE_SLOPE_ARG Dtype min_value,
-#define RELU6_MAXVAL_ARG Dtype max_value,
+#define NEGATIVE_SLOPE_ARG
+#define RELU6_MAXVAL_ARG Dtype min_value, Dtype max_value,
```

or rename this args into "FUSED_ARGS" (`FUSED_CONV_POWER` case is not consistent too)

Reply:
@alalek，thanks for the comment，patch is updated with this change